cPP_DSC.adoc

```diff
@@ -4279,97 +4469,211 @@ FPT_ROT_EXT.1
+
+FCS_CKM.6 - included
 
 |FCS_CKM.1/SKG 
-|[FCS_CKM_EXT.7 or FCS_COP.1] 
+|[FCS_CKM.2, FCS_CKM.5, FCS_CKM_EXT.7 or FCS_COP.1]
+
+[FCS_RBG.1 or FCS_RNG.1]
+
+FCS_CKM.3
+
+FCS_CKM.6 
+|FCS_CKM.2 - included
+
+FCS_CKM.3 is satisfied by FAU_STG_EXT.1 which controls access to keys - included
+
+FCS_CKM.5 - included
+
+FCS_CKM_EXT.7 - (selection-based SFR) included
+
+FCS_COP.1 - included
+
+FCS_RBG.1 - included
+
+FCS_RNG.1 is satisfied by FCS_RBG.1 - included
+
+FCS_CKM.6 - included
+
+|FCS_CKM_EXT.3 
+|[FCS_CKM.1, FCS_CKM.5 or FCS_CKM_EXT.8]
+
+[FCS_COP.1/KeyEncap, FCS_COP.1/KeyWrap, FCS_COP.1/SKC or FCS_COP.1/AEAD]
 
 FCS_CKM.6 
-|All of FCS_CKM_EXT.7, FCS_CKM.6, and FCS_COP.1 are required by the PP.
+|FCS_CKM.1 - included
+
+FCS_CKM.5 - included
+
+FCS_CKM_EXT.8 - (selection-based SFR) included
+
+FCS_COP.1/KeyEncap - (selection-based SFR) included
+
+FCS_COP.1/KeyWrap - (selection-based SFR) included
+
+FCS_COP.1/SKC - (selection-based SFR) included
+
+FCS_COP.1/AEAD - (selection-based SFR) included
+
+FCS_CKM.6 - included
 
 |FCS_CKM.5 
-|FCS_CKM.1 
+|FCS_CKM.2
 
 FCS_COP.1 
 
-FDP_SDC.2 
-|All of FCS_CKM.1, FCS_COP.1, and FDP_SDC.2 are required by the PP.
+FCS_CKM.6
+|FCS_CKM.1 - included
 
-|FCS_CKM_EXT.8 
-|[FDP_ITC.1, FDP_ITC.2] 
+FCS_COP.1 - included
 
-[FCS_CKM.2 or FCS_COP.1 or FCS_CKM_EXT.7]
+FCS_CKM.6 - included
+
+|FCS_CKM_EXT.8 
+|[FCS_CKM.2 or FCS_CKM_EXT.7]
 
 FCS_COP.1/KeyedHash
 
 FCS_CKM.6 
-|FCS_CKM.1/KeyedHash and FCS_CKM.6 are required by the PP. FDP_ITC_EXT.1 and FDP_ITC_EXT.2 are also required by the PP and equivalent to FDP_ITC.1 in this case because they all relate to a mechanism by which key data can be imported into the TSF.
+|FCS_CKM.2 - included
+
+FCS_COP.1 - included
+
+FCS_CKM_EXT.7 - (selection-based SFR) included
+
+FCS_CKM.1/KeyedHash - included
+
+FCS_CKM.6 - included
 
 |FCS_COP.1/AEAD
-|[FDP_ITC.1, FDP_ITC.2, FCS_CKM.1, FCS_CKM.5, FCS_CKM_EXT.7, FCS_CKM_EXT.8]
+|[FDP_ITC.1, FDP_ITC.2, FCS_CKM.1, FCS_CKM.5, FCS_CKM_EXT.7 or FCS_CKM_EXT.8]
+
+FCS_CKM.3
 
 FCS_CKM.6
 
 FCS_OTV_EXT.1
-|FCS_CKM.1 and FCS_CKM.6 are required by the PP. FDP_ITC_EXT.1 and FDP_ITC_EXT.2 are also required by the PP and equivalent to FDP_ITC.1 in this case because they all relate to a mechanism by which key data can be imported into the TSF.
+|FDP_ITC.1 is satisfied by FDP_ITC_EXT.1 related to a mechanism by which key data can be imported into the TSF - included
+
+FDP_ITC.2 is satisfied by FDP_ITC_EXT.2 related to a mechanism by which key data can be imported into the TSF - included
+
+FCS_CKM.1 - included
+
+FCS_CKM.5 - (selection-based SFR) included
+
+FCS_CKM_EXT.7 - (selection-based SFR) included
+
+FCS_CKM_EXT.8 - (selection-based SFR) included
+
+FCS_CKM.3 is satisfied by FAU_STG_EXT.1 which controls access to keys - included
+
+FCS_CKM.6 - included
+
+FCS_OTV_EXT.1 - included
+
+|FCS_COP.1/CMAC
+|[FDP_ITC.1, FDP_ITC.2, FCS_CKM.1, FCS_CKM.5, FCS_CKM_EXT.7 or FCS_CKM_EXT.8]
+
+FCS_CKM.3
+
+FCS_CKM.6
+
+FCS_OTV_EXT.1
```

Comment:
The Crypto Catalog revision that I have includes FCS_COP.1/SigGen here instead of FCS_OTV_EXT.1 but I'm 95% sure that's an error. SigGen wouldn't make sense as a dependency.

Reply:
Remove FCS_OTV_EXT.1

Reply:
removed